packages/gill/package.json

```diff
@@ -111,13 +111,13 @@
   },
   "dependencies": {
     "@gillsdk/config-eslint": "workspace:*",
-    "@solana-program/address-lookup-table": "^0.7.0",
-    "@solana-program/compute-budget": "^0.8.0",
-    "@solana-program/system": "^0.7.0",
-    "@solana-program/token-2022": "^0.4.2",
+    "@solana-program/address-lookup-table": "^0.9.0",
+    "@solana-program/compute-budget": "^0.10.0",
+    "@solana-program/system": "^0.9.0",
+    "@solana-program/token-2022": "^0.6.0",
     "@solana/assertions": "^2.1.1",
```

Comment:
I'd suggest bumping this to `^5.0.0` too. It's part of the Kit monorepo but not part of the `@solana/kit` re-exports. So it does need to be an independent dependency for you to use its exports, but should align with the rest of Kit so you don't end up with multiple versions installed.